docs/changelog.md

```diff
@@ -7,6 +7,8 @@ description: |-
 
 # MCS Provider's changelog
 
+#### v0.6.8
```

Comment:
do rebase please, v0.6.8 has been already realesed and add a line between versions